### PR TITLE
add Elman RNN export to ONNX

### PR DESCRIPTION
**Pull request by anderspapitto**

### CODE REVIEW

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -497,6 +499,41 @@ def reform_weights(g, w, n, intervals):
...
+    if bidirectional:
+        return _unimplemented("RNN", "bidirectional")
+
+    def symbolic(g, input, all_weights, h0, **fkwargs):
+        # TODO elide this argument to increase parametricity. This is
+        # nontrivial because we provide subsequent optional arguments,
+        # and ONNX does not have a mechanism for skipping non-trailing
+        # optional arguments.
+        sequence_len, batch_size = input.type().sizes()[0:2]
+        sequence_lens = g.op('Constant', value_t=torch.IntTensor(batch_size).fill_(sequence_len))
```
Comment (dzhulgakov) [inline torch/onnx/symbolic.py]: btw, we should fix this part of export as well - model should definitely work with different batch sizes (not in this diff of course)

Comment (anderspapitto) [reply torch/onnx/symbolic.py]: yes, agreed

**Comment by ezyang** [comment]
LGTM

**Comment by anderspapitto** [comment]
@ezyang fixed the lint, this is ready to go

**Pull request merged**